.github/workflows/c8run-build.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
           cache: false # disabling since not working anyways without a cache-dependency-path specified
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9 # v8.0.0
```

Comment:
What is that change?

Reply:
The update of this action breaks the linter CI. We rolled back to unblock the release first.